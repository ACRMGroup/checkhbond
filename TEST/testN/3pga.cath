3pga12     3    40    50    40     1     4     1   114 2.00
3pga22     3    40    50    40     1     4     1   114 2.00
3pga32     3    40    50    40     1     4     1   114 2.00
3pga42     3    40    50    40     1     4     1   114 2.00
3pga11     3    40    50  1170     1     2     2   214 2.00
3pga21     3    40    50  1170     1     2     2   214 2.00
3pga31     3    40    50  1170     1     2     2   214 2.00
3pga41     3    40    50  1170     1     2     2   214 2.00
