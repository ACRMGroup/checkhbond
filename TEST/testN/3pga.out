INFO: Processing residue type ARG
INFO: Processing file /acrm/data/dompdb/3pga12
ARG    262 N    : SER    260 OG   H    CB  
INFO: Processing file /acrm/data/dompdb/3pga22
INFO: Processing file /acrm/data/dompdb/3pga32
ARG    262 N    : SER    260 OG   H    CB  
INFO: Processing file /acrm/data/dompdb/3pga42
ARG    262 N    : SER    260 OG   H    CB  
INFO: Processing file /acrm/data/dompdb/3pga11
ARG    127 N    : ASP    135 OD1  H    CG  
INFO: Processing file /acrm/data/dompdb/3pga21
Warning=> genh() found no carbonyl carbon preceeding residue 222 
Warning=> genh() found no carbonyl carbon preceeding residue 231 
Warning=> genh() found no carbonyl carbon preceeding residue 235 
Warning=> genh() found no carbonyl carbon preceeding residue 236 
